moped-editor/src/views/auth/LoginView.js

```diff
@@ -173,6 +171,11 @@ const LoginView = () => {
               onClick={() => signIn()}
               size="large"
               variant="contained"
+              disabled={
+                !["staging", "production"].includes(
+                  process.env.REACT_APP_HASURA_ENV
+                )
+              }
```

Comment:
SSO is not supported in netlify or local, so we should just disable the button, right?